### DISCUSSION TITLE
How secured is polygon edge's chainbridge?

### DISCUSSION THREAD

**Discussion by taylorkchan** [0xPolygon/polygon-edge#554] [General]
Hi, our team is considering to use polygon edge to build our own sidechain. We see that polygon edge provides chainbridge  and the relayer contracts. How secured and are there any security concerns using this instead of other bridging solutions like Celer?

**Comment by laviniat1996** [comment]
Hi @taylorkchan . Thank you for your question and interest :)

Chainbridge is a **relayer type** bridge.

You can achieve a higher level of security by operating multiple relayers which need to vote on a certain action taking place.
But, if a malicious party is to get access to the   private keys of the majority of the relayers, the bridge is compromised.

Chainbridge is **audited**, and as such is treated as secure.
